src/AccessibilityInsights.SharedUx/Behaviors/DropDownButtonBehavior.cs

```diff
@@ -52,17 +43,49 @@ private void AssocisatedObject_ContextMenuOpening(object sender, ContextMenuEven
             e.Handled = true;
         }
 
+        private void ForContainedFabricIconControls(Action<FabricIconControl> action)
+        {
+            if (_containedFabricIconControls != null)
+            {
+                foreach (FabricIconControl fabricIconControl in _containedFabricIconControls)
+                {
+                    action(fabricIconControl);
+                }
+            }
+        }
+
+        private void StyleContainedFabricIconControls()
+        {
+            _containedFabricIconControls = ContainedControlFinder<FabricIconControl>.Find(AssociatedObject).ToList();
+
+            if (_containedFabricIconControls.Any())
+            {
+                // This assumes that all contained FabricIconControls use the same style
+                _savedFabricIconControlStyle = _containedFabricIconControls[0].Style;
+                Style pressedStyle = AssociatedObject.FindResource("fabricIconOnPressedButtonParentStyle") as Style;
```

Comment:
I tried various combinations of setting the Foreground on the FabricIconControl, but couldn't get it to work. I finally went with replacing the Style. This will have side effects if the FabricIconControl uses a style for anything besides colors